src/main/java/use_cases/main_menu/SessionSaveUCI.java

```diff
@@ -0,0 +1,33 @@
+package use_cases.main_menu;
+
+import use_cases.session_load.SessionStorageG;
```

Comment:
Gateways are a part of the interface adapters layer, so this should be in the views package. And the UCI should not know it exists.
<img width="645" alt="Screenshot 2023-08-04 at 2 39 21 PM" src="https://github.com/CSC207-2023Y-UofT/course-project-DoorKeepers/assets/125835221/ea52d41e-27a9-471a-9670-9b578a81ae53">

Reply:
You will likely need to add some sort of Gateway Boundary between the two, to act like the OB.

Reply:
Please ignore me, I forgot your SessionStorageG was the boundary!